Sources/MapboxMaps/Location/LocationManager.swift

```diff
@@ -11,7 +11,7 @@ public final class LocationManager: NSObject {
     }
 
     /// The object that acts as the delegate of the location manager.
-    public weak var delegate: LocationPermissionsDelegate?
+    public weak var delegate: LocationManagerDelegate?
```

Comment:
This is a breaking change.
I suppose your workaround with `LocationPermissionsDelegate: LocationManagerDelegate` is still a breaking change because the delegate property could be tested for being the concrete type `is LocationPermissionsDelegate`. Unfortunately, we have to keep the `LocationPermissionsDelegate` type here and cut the renaming ticket for the next release.

Reply:
Indeed, this is technically a breaking change. Practically I figure this should be fine as this is non-source breaking change. Behavior-wise it should be work as well - if somebody is going to check if `delegate is LocationPermissionsDelegate` then they set it previously to `LocationPermissionsDelegate`(since they still use the old API).

Reply:
![image](https://user-images.githubusercontent.com/30261043/183892945-16a3c213-da65-4283-9ba4-bdb5292e986e.png)
I agree with @OdNairy on this and cut the renaming for next major release. Even though it is silly to check against concrete `LocationPermissionsDelegate`, there is still a possibility and thus this change will be breaking changes for them.